src/test/java/StringTest.java

```diff
@@ -0,0 +1,40 @@
+import org.assertj.core.api.AbstractThrowableAssert;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("String class split test")
+    void splitTest() {
+        String[] actual = "1,2".split(",");
+        assertThat(actual).containsExactly("1", "2");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?